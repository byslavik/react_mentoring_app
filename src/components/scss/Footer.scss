.footerWrapper {
  background: #ccc;
}
.footer {
  max-width: 1000px;
  padding: 20px 0;
}