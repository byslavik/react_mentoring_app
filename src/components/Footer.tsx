import * as React from 'react';

import * as s from './Footer.scss';

export class Footer extends React.Component {

    render() {
        return (<div className={s.footerWrapper}>
          <footer className={s.footer}>

          </footer>
        </div>)
    }
}
