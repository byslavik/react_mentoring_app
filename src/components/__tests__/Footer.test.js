import * as React from 'react';
import {Footer} from '../Footer';
import * as renderer from 'react-test-renderer';


describe('Footer component', ()=>{
  it('Footer component shows correctly', () => {
    const tree = renderer.create(
      <Footer />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
})