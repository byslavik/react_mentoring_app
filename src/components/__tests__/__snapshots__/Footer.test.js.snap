// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Footer component shows correctly 1`] = `
<div
  className={undefined}
>
  <footer
    className={undefined}
  />
</div>
`;
